### PR TITLE
SEO-176496-xamarin-page-with-redirect-links

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Please review the changes.

Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/Tabbed-View/Getting-Started.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Location: {Installed location}/{version}/Xamarin/lib
 </tr>
 </table>
 
-N> To know more about obtaining our components, refer to these links for [Mac](https://help.syncfusion.com/xamarin/introduction/download-and-installation/mac/) and [Windows](https://help.syncfusion.com/xamarin/introduction/download-and-installation/windows/).
+N> To know more about obtaining our components, refer to these links for [Mac](https://help.syncfusion.com/xamarin/installation/mac-installer/how-to-download) and [Windows](https://help.syncfusion.com/xamarin/installation/web-installer/how-to-downloads).
```
Comment (Yvone-Atieno) [inline Xamarin/Tabbed-View/Getting-Started.md:63]: Check this link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Tabbed-View/Getting-Started.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/Tabbed-View/Getting-Started.md:63]: updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**